app/src/components/layout/navLinks/NavLinks.tsx

```diff
@@ -8,25 +8,74 @@ import { useSessionContext } from '../../../providers/sessionProvider/SessionPro
 const NavLinks = () => {
     const navigate = useNavigate();
     const [session] = useSessionContext();
-
+    const [width, setWidth] = useState<number>(window.innerWidth);
     const nav = (e: ReactEvent<HTMLAnchorElement, MouseEvent>, link: string) => {
         e.preventDefault();
         navigate(link);
     };
 
+    console.log(width);
+    useEffect(() => {
+        const handleResizeWindow = () => setWidth(window.innerWidth);
+        window.addEventListener('resize', handleResizeWindow);
+        return () => {
+            window.removeEventListener('resize', handleResizeWindow);
+        };
+    }, []);
+
     return session.isLoggedIn ? (
-        <Header.Nav>
-            <Header.NavItem role="link" className="clickable" onClick={(e) => nav(e, routes.HOME)}>
-                Home
-            </Header.NavItem>
+        <Header.Nav style={{ display: 'initial', color: 'white' }}>
```

Comment:
Not directly related to this line, but within this Header.Nav component they got an embedded close button which only appears when width < 990 (see below pic)
![image](https://github.com/nhsconnect/national-document-repository/assets/127404525/63be9677-a0df-40bd-aadb-4e02cfd034df)

As now we override the css and make this Nav to appear all the time, I would suggest to override the class "nhsuk-header navigation-close" to display:none, so that we don't have a close button that doesn't serve any purpose